app/models/card.rb

```diff
@@ -0,0 +1,9 @@
+class Card < ActiveRecord::Base
+	belongs_to :deck
+	has_many :guesses
+	has_many :rounds, through: :guesses
+
+	validates :question, null: false, uniqueness: { scope: :deck }
+	validates :answer, null: false
+	validates :deck_id, null: false
```

Comment:
if we use `validates_presence of :deck` we can validate that not only there is a deck_id, but that there is a deck associated with that deck_id